TruexGoogleReferenceApp/AppDelegate.swift

```diff
@@ -40,5 +47,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
     }
 
+    func initializeCrashlytics() {
+        Fabric.with([Crashlytics.self])
 
+        #if DEBUG
```

Comment:
Super-Super-nit: feels like this would be a little more readable with indentation. Though not sure what the convention here is